src/utils/env/index.ts

```diff
@@ -1,8 +1,11 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import bytes from 'bytes';
-import { Duration } from 'luxon';
+import dayjs from 'dayjs';
+import duration, { Duration } from 'dayjs/plugin/duration';
 import ms from 'ms';
 
+dayjs.extend(duration);
+
 const booleanFalseValues = ['false', '0', 'no', 'off'];
 
 export class Env {
```

Comment:
```suggestion
export class Env {
	static getString(key: string, default: string = ''): string {
		return process.env[key] || default;
	}
```

I think it would be better to have the option to pass a default value in the case of the environment variable is not present and let this method handle that.